projects/node-api/src/Client.ts

```diff
@@ -73,14 +73,18 @@ export class Client {
     constructor (
         username: string, password: string,
         protocol = 'https', host = 'hub.graphistry.com',
-        clientProtocolHostname?: string
+        clientProtocolHostname?: string,
+        previousClient?: Client
     ) {
         this.username = username;
         this._password = password;
         this.protocol = protocol;
         this.host = host;
         this.clientProtocolHostname = clientProtocolHostname || `${protocol}://${host}`;
-        this.getAuthToken();
+        if (previousClient && username === previousClient.username && password === previousClient._password && protocol === previousClient.protocol) {
+            this._token == previousClient._token;
+        }
```

Comment:
surprising typescript doesn't throw a compile error/warning as `_token` is private

Reply:
actually... we can revert all these changes, right? we're not supposed to do anything with `node-api`